### PR TITLE
Better release instructions

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Update the readme to add a missing publishing process steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,10 +99,21 @@ Prerequisites:
...
+$ echo "//registry.npmjs.org/:_authToken=YOUR-ACCESS-TOKEN" > ~/.npmrc
+```
+
+replacing YOUR-ACCESS-TOKEN with your own access token
 
 Then:
 
-`yarn release --patch` or `yarn release --minor` or `yarn release --major`
+```
+$ yarn release
```
Comment (arelra) [inline README.md:113]: Even though you select the type interactively could the release type be used elsewhere by the publish script?

**Review by arelra — APPROVED** [review]

**Comment by mxdvl** [comment]
Good to add info on this. There are a few auto-published NPM libraries such as [@guardian/libs], [@guardian/commercial-core] and [@guardian/consent-management-platform]. It might be worth looking into incorporating `semantic-release` here, too? Maybe something @guardian/client-side-infra can help with?

[@guardian/libs]: https://github.com/guardian/libs/
[@guardian/consent-management-platform]: https://github.com/guardian/consent-management-platform/
[@guardian/commercial-core]: https://github.com/guardian/commercial-core/

File: README.md (hunk 1 above)
Comment (shtukas) [reply README.md:113]: 🤔
Let me change that.... (I mean give the three options), we will figure out later if `yarn release` is enough.

**Comment by shtukas** [comment]
@mxdvl 
Nice one. Added to todo list :)

**Pull request merged**